packages/webview_flutter/webview_flutter_android/lib/src/android_proxy.dart

```diff
@@ -40,7 +40,8 @@ class AndroidWebViewProxy {
     Future<List<String>> Function(
       android_webview.WebView webView,
       android_webview.FileChooserParams params,
-    )? onShowFileChooser,
+    )?
```

Comment:
Your formatting is most likely different because you are running the formatter on a older version of Flutter. We develop plugins on the the main branch of Flutter. So you may need to change to the latest channel before running the formatter or fix these manually.